time_tracker_api/customers/customers_namespace.py

```diff
@@ -45,7 +46,7 @@ class Customers(Resource):
     @ns.marshal_list_with(customer)
     def get(self):
         """List all customers"""
-        return customer_dao.get_all()
+        return customer_dao.get_all(conditions=request.args)
```

Comment:
What if you have some query like this?
```
/activities?a=1&b=2&pageSize=0&orders["start_date"]="DESC"&limit=1
```
This is not going to work

Reply:
In addition, it would be nice to filter per namespace that the attributes you want to filter by are valid.

Reply:
Good catch

Reply:
> What if you have some query like this?
> 
> ```
> /activities?a=1&b=2&pageSize=0&orders["start_date"]="DESC"&limit=1
> ```
> 
> This is not going to work, because `a` and `b` are the attributes but you are filtering for whatever it is specified in the request arguments.

Do we have filtering and sorting arguments right now?

Reply:
The repository has sorting capabilities right now. For instance, The repository for time entries orders the results by `start_date`. We could do this dynamically in the future.